web-portal/frontend/fly.staging.toml

```diff
@@ -7,20 +7,22 @@ app = 'porters-frontend-staging'
 primary_region = 'sea'
 
 [build.args]
-  NEXT_PUBLIC_API_ENDPOINT = 'https://porters-backend-staging.fly.dev/'
+    API_ENDPOINT = 'https://porters-backend-staging.fly.dev/'
 
 [http_service]
-  internal_port = 3000
-  force_https = true
-  auto_stop_machines = true
-  auto_start_machines = true
-  min_machines_running = 0
-  processes = ['app']
+    internal_port = 3000
+    force_https = true
+    auto_stop_machines = true
+    auto_start_machines = true
+    min_machines_running = 0
+    processes = ['app']
 
 [env]
-  NEXT_PUBLIC_API_ENDPOINT = 'https://porters-backend-staging.fly.dev/'
+    NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID = 'wallet-connect-project-id'
+    NEXT_PUBLIC_RPC_ENDPOINT = 'rpc-endpoint'
```

Comment:
This is actually setting the environment variables to this, they get overwritten by secrets, but it isn't clear that this is a great way to document the need for the secrets. Not sure though.

Reply:
I agree but we shouldn't put our secrets in repo anyway

Reply:
Right, but this doesn't do anything right here, so a bit confusing.